client/src/components/admin/AdminCache.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div id="adminCachePanel" class="container py-3">
+    <div class="col-6 col">
+      <div>
+        <h5>Cache bereinigen</h5>
```

Comment:
Ist zwar nur für uns aber ein Dropdown mit Optionen wäre benutzerfreundlicher

Reply:
Er löscht alle Keys in denen der Wert aus dem Feld vorkommt.

z.B. löscht "result" folgende Einträge falls es sie gibt
/result?year=2042
/result?year=2042&foo=bar

Deshalb macht dann ein Dropdown keinen Sinn.

Vielleicht braucht man hier ein besseres Label. Aber wie kann man das besser bezeichnen?